airflow/providers/celery/sensors/celery_queue.py

```diff
@@ -73,5 +72,9 @@ def poke(self, context: Context) -> bool:
             self.log.info("Checking if celery queue %s is empty.", self.celery_queue)
 
             return reserved == 0 and scheduled == 0 and active == 0
-        except KeyError:
-            raise KeyError(f"Could not locate Celery queue {self.celery_queue}")
+        except Exception as err:
+            # TODO: remove this if check when min_airflow_version is set to higher than 2.7.1
+            message = f"Could not locate Celery queue {self.celery_queue}"
+            if self.soft_fail:
+                raise AirflowSkipException(message) from err
+            raise AirflowException(message) from err
```

Comment:
IMHO we should keep the `except KeyError:` and add a new except for the other Exception, to keep the exact same behavior and just fix the bug